ui/src/shared/components/cells/Cell.scss

```diff
@@ -173,6 +177,12 @@ $cell--header-size: 36px;
   }
 }
 
+.vis-plot-container {
+  width: 100%;
+  height: 100%;
+  padding: 15px;
```

Comment:
Same comment about variables, there's one that's `16px` I think it's `$ix-marg-c`